_includes/v22.2/ui/sessions.md

```diff
@@ -6,24 +6,27 @@ To filter the sessions, click the **Filters** field.
 
 <img src="{{ 'images/v22.2/ui-session-filter.png' | relative_url }}" alt="Session filter" />
 
-To filter by [application]({{ link_prefix }}connection-parameters.html#additional-connection-parameters), select **App** and choose one or more applications.
+- To filter by [application]({{ link_prefix }}connection-parameters.html#additional-connection-parameters), select **App** and choose one or more applications.
 
-- Queries from the SQL shell are displayed under the `$ cockroach` app.
-- If you haven't set `application_name` in a client connection string, it appears as `unset`.
+    - Queries from the SQL shell are displayed under the `$ cockroach` app.
+    - If you haven't set `application_name` in a client connection string, it appears as `unset`.
 
-To filter by session duration, specify the session time and unit.
+- To filter by username or session status, select **Username** or **Session Status** and choose one or more checkboxes.
```

Comment:
Nit: It feels a little weird to "choose" a checkbox, but I grant that "check" may be redundant. Maybe "fill" or "click"?

```suggestion
- To filter by username or session status, select **Username** or **Session Status** and check one or more checkboxes.
```